cloud/azure/modules/app/main.tf

```diff
@@ -63,12 +62,18 @@ resource "azurerm_service_plan" "plan" {
   sku_name            = var.app_sku
 }
 
+resource "azurerm_app_service_virtual_network_swift_connection" "app_vn" {
+  app_service_id = azurerm_linux_web_app.civiform_app.id
+  subnet_id      = azurerm_subnet.server_subnet.id
+}
+
 resource "azurerm_linux_web_app" "civiform_app" {
-  name                = "${var.application_name}-${random_pet.server.id}"
-  location            = data.azurerm_resource_group.rg.location
-  resource_group_name = data.azurerm_resource_group.rg.name
-  service_plan_id     = azurerm_service_plan.plan.id
-  app_settings        = local.app_settings
+  name                      = "${var.application_name}-${random_pet.server.id}"
+  location                  = data.azurerm_resource_group.rg.location
+  resource_group_name       = data.azurerm_resource_group.rg.name
+  service_plan_id           = azurerm_service_plan.plan.id
+  app_settings              = local.app_settings
+  virtual_network_subnet_id = azurerm_subnet.server_subnet.id
```

Comment:
**question:** This was the main change since https://github.com/civiform/cloud-deploy-infra/pull/366 that did it, right?

Reply:
yep!